check.go

```diff
@@ -51,7 +51,7 @@ func CheckDir(synta synta.Synta, fs fs.FS, dirPath string, recursive bool, ensur
 			err = fmt.Errorf("Could not read directory: %v", err)
 			return err
 		}
-		log.Info().Msg("Checking file/dir with name " + file.Name())
+		log.Info().Msg("Checking file/dir with name " + dirPath + "/" + file.Name())
```

Comment:
Qui lo vogliamo lasciare? Perche' lo stampa in ogni caso (come con `go test`)

Reply:
Potremmo usare il logging solo ed esclusivamente dentro a `linter.go` e passare queste informazioni in qualche modo, in realta' forse questi log possiamo proprio toglierli

Reply:
Risolto no?